### PR TITLE
doc/start/os-recommendations.rst: add Octopus' platform

### PR DESCRIPTION
**Pull request by JackSlateur**
This should also be backported into the octopus branch

Signed-off-by: Alexandre Bruyelles <jack@jack.fr.eu.org>

### CODE REVIEW

**Comment by tchaikov** [comment]
jenkins render docs

File: doc/start/os-recommendations.rst [hunk 1]
```diff
@@ -39,6 +39,25 @@ platforms.  Generally speaking, there is very little dependence on
...
++----------+----------+--------------------+--------------+---------+------------+
+| Distro   | Release  | Code Name          | Kernel       | Notes   | Testing    |
++==========+==========+====================+==============+=========+============+
+| CentOS   | 8        | N/A                | linux-4.18   |         | B, I, C    |
++----------+----------+--------------------+--------------+---------+------------+
+| Centos   | 7        | N/A                | linux-3.10.0 | 3       | B, I,      |
++----------+----------+--------------------+--------------+---------+------------+
+| Debian   | 10.0     | Buster             | linux-4.19   |         | B, I       |
++----------+----------+--------------------+--------------+---------+------------+
+| RHEL     | 8        | Ootpa              | linux-4.18   |         | B, I       |
```
Comment (tchaikov) [inline doc/start/os-recommendations.rst:54]: i think we should add "C" to this list.

File: doc/start/os-recommendations.rst [hunk 2]
```diff
@@ -39,6 +39,25 @@ platforms.  Generally speaking, there is very little dependence on
...
 
+Octopus (15.2.z)
+-----------------
+
++----------+----------+--------------------+--------------+---------+------------+
+| Distro   | Release  | Code Name          | Kernel       | Notes   | Testing    |
++==========+==========+====================+==============+=========+============+
+| CentOS   | 8        | N/A                | linux-4.18   |         | B, I, C    |
++----------+----------+--------------------+--------------+---------+------------+
+| Centos   | 7        | N/A                | linux-3.10.0 | 3       | B, I,      |
```
Comment (tchaikov) [inline doc/start/os-recommendations.rst:50]:
```suggestion
| CentOS   | 7        | N/A                | linux-3.10.0 | 3       | B, I,      |
```

File: doc/start/os-recommendations.rst [hunk 3]
```diff
@@ -39,6 +39,25 @@ platforms.  Generally speaking, there is very little dependence on
...
++----------+----------+--------------------+--------------+---------+------------+
+| Debian   | 10.0     | Buster             | linux-4.19   |         | B, I       |
```
Comment (tchaikov) [inline doc/start/os-recommendations.rst:52]: probably should remove "I", as we don't perform functional tests on Debian buster.

File: doc/start/os-recommendations.rst (hunk 2 above)
Comment (tchaikov) [inline doc/start/os-recommendations.rst:50]: please drop "3" from the notes column, we don't test with btfs anymore.

File: doc/start/os-recommendations.rst [hunk 4]
```diff
@@ -39,6 +39,25 @@ platforms.  Generally speaking, there is very little dependence on
...
++----------+----------+--------------------+--------------+---------+------------+
+| RHEL     | 8        | Ootpa              | linux-4.18   |         | B, I       |
++----------+----------+--------------------+--------------+---------+------------+
+| RHEL     | 7        | Maipo              | linux-3.10.0 |         | B, I       |
++----------+----------+--------------------+--------------+---------+------------+
+| Ubuntu   | 18.04    | Bionic Beaver      | linux-4.15   | 3       | B, I, C    |
```
Comment (tchaikov) [inline doc/start/os-recommendations.rst:58]: please drop "3" from the notes column, we don't test with btfs anymore.

File: doc/start/os-recommendations.rst (hunk 2 above)
Comment (JackSlateur) [reply doc/start/os-recommendations.rst]: Thanks, I've clarified the btrfs situation on Octopus

**Comment by neha-ojha** [comment]
jenkins render docs

File: doc/start/os-recommendations.rst [hunk 5]
```diff
@@ -39,6 +39,25 @@ platforms.  Generally speaking, there is very little dependence on
...
++----------+----------+--------------------+--------------+---------+------------+
+| Debian   | 10       | Buster             | linux-4.19   |         | B          |
++----------+----------+--------------------+--------------+---------+------------+
+| RHEL     | 8        | Ootpa              | linux-4.18   |         | B, I, C    |
++----------+----------+--------------------+--------------+---------+------------+
+| RHEL     | 7        | Maipo              | linux-3.10.0 |         | B, I       |
++----------+----------+--------------------+--------------+---------+------------+
+| Ubuntu   | 18.04    | Bionic Beaver      | linux-4.15   | 4       | B, I, C    |
++----------+----------+--------------------+--------------+---------+------------+
+
```
[the remark this answers is not in the document]
Comment (tchaikov) [reply doc/start/os-recommendations.rst:60]: @neha-ojha ping?

[the remark this answers is not in the document]
Comment (neha-ojha) [reply doc/start/os-recommendations.rst:60]: I'll defer this to @capri1989 who added it in https://github.com/ceph/ceph/pull/28643

[the remark this answers is not in the document]
Comment (JackSlateur) [reply doc/start/os-recommendations.rst:63]: @tchaikov @neha-ojha I've updated the MR accordingly
Also, I added a note on centos7's support in Octopus

**Comment by JackSlateur** [comment]
jenkins render docs

**Pull request merged**